src/main/java/io/xpring/payid/XRPPayIDClient.java

```diff
@@ -0,0 +1,34 @@
+package io.xpring.payid;
+
+import io.xpring.common.XRPLNetwork;
+
+/**
+ * Provides functionality for XRP in the PayID protocol.
+ */
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+public class XRPPayIDClient extends PayIDClient implements XRPPayIDClientInterface {
+  /**
+   * The XRP Ledger network that this client attaches to.
+   */
+  private XRPLNetwork xrplNetwork;
+
+  /**
+   * Get the XRP Ledger network that this client attaches to.
+   *
+   * @return The XRP Ledger network that this client attaches to.
+   */
+  public XRPLNetwork getXRPLNetwork() {
+    return this.xrplNetwork;
+  }
+
+  /**
+   * Construct a new XRPPayIDClient.
+   *
+   * @param xrplNetwork The XRP Ledger network that this client attaches to.
+   */
+  public XRPPayIDClient(XRPLNetwork xrplNetwork) {
+    super("xrpl-" + xrplNetwork.getNetworkName());
+
+    this.xrplNetwork = xrplNetwork;
+  }
+}
```

Comment:
should this class implement `xrpAddressForPayID` ?

Reply:
(and PayIDClient should have a method just called `addressForPayID` ? )  Or maybe all happening in subsequent PRs?

Reply:
It will in the future.